.github/workflows/autoconf-archive.yml

```diff
@@ -1,22 +1,23 @@
+---
```

Comment:
What linter is throwing a warning about not having an opening YAML separator in GH workflow files? This just isn't a thing.

Reply:
`yamllint` is

Reply:
This is a document separator and only relevant in the case where multiple YAML documents may be included in a single file. Such is not the case for GH workflow definitions and it is relatively rare to do so. Not invalid per se but not recommended by GH and typically not used because it is optional and could never be meaningful in this context.

Reply:
OK, I disabled the warning instead in [3c4cecd](https://github.com/autoconf-archive/autoconf-archive/pull/310/commits/3c4cecdea0c0a00d861252e82e6f4b3919017313).